CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 ### Dependencies
 1. [#472](https://github.com/influxdata/influxdb-client-python/pull/472): Update `RxPY` to `4.0.4`
 
+### Bug Fixes
+1. [#485](https://github.com/influxdata/influxdb-client-python/issues/485): Fix bug when serializing DataFrames that might occur if you're inserting NaN values and have columns starting with digits.
+
```

Comment:
Please move this to the existing `Bug Fixes` section:

![image](https://user-images.githubusercontent.com/455137/183426648-a95208bd-2afa-4c98-9db7-9c8fdedd6ccf.png)

Reply:
I've moved it to the existing section 👍 . Completely missed that during the rebase. I've also updated it to point at the PR instead of the issue.

Reply:
Now it's ok, thanks.